CHANGELOG.md

```diff
@@ -21,11 +21,16 @@ should change the heading of the (upcoming) version to include a major version b
 ## @rjsf/core
 
 - Fix default value population when switching between options in `MultiSchemaField` [#4375](https://github.com/rjsf-team/react-jsonschema-form/pull/4375). Fixes [#4367](https://github.com/rjsf-team/react-jsonschema-form/issues/4367)
+- Updated `ObjectField` and `CheckboxWidget` to render markdown in the description when `enableMarkdownInDescription` is set to `true`, fixing [#3975](https://github.com/rjsf-team/react-jsonschema-form/issues/3975)
 
 ## @rjsf/validator-ajv8
 
 - Fixed issue where `ui:title` in anyOf/oneOf is not shown in error messages. Fixes [#4368](https://github.com/rjsf-team/react-jsonschema-form/issues/4368)
 
+## Dev / docs / playground
+
+- Fixed typo in `package.json`
+
```

Comment:
The package change is so small as to not require mentioning. What does want to be mentioned is that it is a new feature so requires `5.25.0` AND you want to mention the changes in each package.